.idea/misc.xml

```diff
@@ -49,7 +56,7 @@
   <component name="ProjectResources">
     <default-html-doctype>http://www.w3.org/1999/xhtml</default-html-doctype>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="false" assert-keyword="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="false" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```

Comment:
Looks good overall, and generally it's ok to sync IDE project files and commit them. I'm just wondering about this one removal of assert-keyword="true". I'm going to try to get my IDE to do that locally. I just want to make sure it doesn't mean we run tests with the assert keyword disabled (by removing the -ea flag).